static/js/admin/vue-permissions-editor/src/components/patronRoles.vue

```diff
@@ -1,5 +1,8 @@
 <template>
     <div id="patron-roles">
+        <p class="duracloud-note">Access control changes here will not change settings in Duracloud, any corresponding changes must
```

Comment:
This note would only need to display for fileobjects. I am also wondering if we should hold off on adding this note for now since we may not end up following the same access control model for streaming stuff in boxc